bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php

```diff
@@ -364,6 +364,11 @@ private function responseCanBeCached(Response $response): bool
     {
         $cache = true;
 
+        // do not cache when the application indicated a 'no-store' in the response Cache-Control header
+        if ($response->headers->getCacheControlDirective('no-store')) {
```

Comment:
Should we also respect "private" and "no-cache" headers here?

Reply:
Good one, added those as well